assets/javascripts/src/components/Main.jsx

```diff
@@ -130,8 +129,8 @@ class Main extends React.Component {
             </MediaQuery>
         </div>
     }
+
     renderSummary(visible) {
```

Comment:
There is some useless code left over by my previous pull request.
There is no need for renderSummary anymore since both the mobile and desktop cases are the same now.
I can fix this is another pull request or I can push a fix here if you want